src/pages/LoadUnload/components/MapLayers/MapLayers.tsx

```diff
@@ -1,19 +1,32 @@
+import { useEffect } from 'react'
+
 import { BaseLayer } from '@amsterdam/arm-core'
-import { TileLayer } from '@amsterdam/react-maps'
+import { TileLayer, useMapInstance } from '@amsterdam/react-maps'
 import { HighlightedFeatureLayer } from 'shared/components/HighlightedFeatureLayer'
 import { AddressMarker } from 'shared/components/MapLayers/AddressMarker'
 import { oneWayArrows, topoBlackWhite } from 'shared/map/mapLayers'
 
-import { useLoadUnloadMapContext } from '../../contexts/MapContext'
+import useLoadUnloadMapContext, { MapLayerId } from '../../contexts/MapContext'
 import { useLoadUnloadPageContext } from '../../contexts/PageContext'
 
+import BollardsLayer, { BOLLARDS_ZOOM_LEVEL } from './BollardsLayer'
 import { LoadUnloadLoadUnloadSpacesLayer } from './LoadUnloadSpacesLayer'
 import { LoadUnloadRoadSectionsLoadUnloadLayer } from './RoadSectionsLoadUnloadLayer'
 
 export const LoadUnloadMapLayers = () => {
-  const { detailFeature } = useLoadUnloadMapContext()
+  const { detailFeature, updateActiveMapLayers } = useLoadUnloadMapContext()
   const { address } = useLoadUnloadPageContext()
 
+  const mapInstance = useMapInstance()
+  useEffect(() => {
+    mapInstance.on('zoomend', () => {
+      updateActiveMapLayers({
+        type: mapInstance.getZoom() >= BOLLARDS_ZOOM_LEVEL ? 'ON' : 'OFF',
```

Comment:
- In the reducer function there are, currently, 3 different action types: ON, OFF and TOGGLE. It refers to that.
- For leaflet it works like this: the higher the zoom level (number) the closer you are to the ground level.
- Thanks!

Reply:
- [ ] My bad, I wasn't clear. My question was more regarding the name of the property.  Isn't `type` too generic and unspecific a name for this behaviour?
- [x] okay clear. It isn't clear to me because I am not too familiar with the application but if you are, it is clear! Would probably be a bit clearer with `BOLLARDS_MIN_ZOOM_LEVEL` but I wouldn't make the effort to change it

Reply:
- Yeah, maybe the naming is not good, but it is pre-existing code, and I'm not going to refactor too much, right now.
- No problem! It's better to be too critical and question things than to be indifferent.